src/bp/core/routers/util.ts

```diff
@@ -108,21 +98,35 @@ export const loadUser = (authService: AuthService) => async (req: Request, res:
 
 /**
  * This method checks that uploaded file respects constraints
- * TODO add * in allowedMimeTypes ==> accepts all
+ * @example fileUploadMulter(['image/*', 'audio/mpeg'], '150mb)
+ * fileUploadMulter(['*'], '1gb)
  */
-export const fileUploadMulter = (allowedMimeTypes: string[] = [], maxFileSize?: string) =>
-  multer({
+export const fileUploadMulter = (allowedMimeTypes: string[] = [], maxFileSize?: string) => {
+  const allowedMimeTypesRegex = allowedMimeTypes.map(mimeType => {
+    // '*' is not a valid regular expression
+    if (mimeType === '*') {
+      mimeType = '.*'
+    }
+
+    return new RegExp(mimeType, 'i')
+  })
+
+  return multer({
     fileFilter: (_req, file, cb) => {
       const extMimeType = mime.lookup(file.originalname)
-      if (allowedMimeTypes.includes(file.mimetype) && allowedMimeTypes.includes(extMimeType)) {
+      if (
+        allowedMimeTypesRegex.some(regex => regex.test(file.mimetype)) &&
+        extMimeType &&
```

Comment:
why add this ? is there any possibility that extMimeType to be undefined ?

Reply:
`mime.lookup` returns either a string or a boolean (false when no mime-type is found). That is why I do the check.